pontoon/urls.py

```diff
@@ -40,6 +41,9 @@
     url(r'^pontoon\.js$',
         RedirectView.as_view(url='/static/js/pontoon.js', permanent=True)),
 
+    # Robots.txt
+    (r'^robots.txt$', lambda r: HttpResponse("User-agent: *\nDisallow: /a/\nDisallow: /admin/", content_type="text/plain")),
```

Comment:
@Osmose Added in one line as it will not be used a lot, but yes, I can change it to template if needed.